dotfiles/.pa11yci.js

```diff
@@ -52,7 +52,7 @@ const config = {
 		},
 		timeout: 50000,
 		wait: process.env.PA11Y_WAIT || 300,
-		hideElements: 'iframe[src*=google],iframe[src*=proxy]',
+		hideElements: 'iframe[src*=google],iframe[src*=proxy],iframe[src*=doubleclick],',
```

Comment:
Not sure how this string get's interpreted so as the previous didn't have a trailing comma I think it's safest to remove.

```suggestion
		hideElements: 'iframe[src*=google],iframe[src*=proxy],iframe[src*=doubleclick]',
```